crates/services/gas_price_service/src/v1/service.rs

```diff
@@ -152,12 +152,12 @@ where
         }
     }
 
-    #[cfg(test)]
+    #[cfg(any(test, feature = "test-helpers"))]
```

Comment:
Nit: Isn't it sufficient to just feature gate on "test-hepers" and ensure the flag is configured in the tests?.